empress/support_files/js/empress.js

```diff
@@ -2487,8 +2545,11 @@ define([
 
         // color tree
         this._colorTree(obs, cm);
-
-        this.updateLegendCategorical(cat, keyInfo);
+        if (continuous) {
+            this._legend.addContinuousKey(cat, keyInfo);
```

Comment:
```suggestion
            this.resizeLegend();
            this._legend.addContinuousKey(cat, keyInfo);
```

(now that #512 is merged in)